src/test/kotlin/org/opensearch/replication/MultiClusterRestTestCase.kt

```diff
@@ -253,7 +253,33 @@ abstract class MultiClusterRestTestCase : OpenSearchTestCase() {
      */
     @Before
     fun setup() {
-        testClusters.values.forEach { if(it.securityEnabled && !it.defaultSecuritySetupCompleted) setupDefaultSecurityRoles(it) }
+        testClusters.values.forEach {
+            registerSnapshotRepository(it)
```

Comment:
Can this inflate running time of tests since it applies to all? If so, can we special case it to where we need?

Reply:
This should result in one cluster state update call, which shouldn't increase the running time (and is same as other operations we are performing for clean-up).

Reply:
Thanks for checking on the perf angle. I think we should be ok, but do consider abstracting the repo name into a variable instead of hard-coding.